docs/docs-beta/src/styles/custom.scss

```diff
@@ -605,3 +605,39 @@ dd code {
     }
   }
 }
+
+
+/**
+ * api docs: inline flag
+ *
+ * available options:
+ * - info
+ * - danger
+ * - warning
+ */
+
+span {
+  &.flag {
+    font-weight: var(--ifm-font-weight-semibold);
+    border-radius: 8px;
+    padding-left: 4px;
+    padding-right: 4px;
+  }
+
+  &.flag-info {
+    background-color: var(--theme-color-background-cyan);
+    color: var(--theme-color-text-cyan);
+
+  }
+
+  &.flag-danger {
+    background-color: var(--theme-color-background-red);
+    color: var(--theme-color-text-red);
+  }
+
+  &.flag-warning {
+    background-color: var(--theme-color-background-yellow);
+    color: var(--theme-color-text-yellow);
+  }
+
+}
```

Comment:
@braunjj you can likely improve the aesthetics here. Current implementation wraps flags in a span with these classes. Alternatively we could use a React component to support things like hover tooltips.
```
<span class="flag flag-warning">experimental</span>
```